source/content/apcu.md

```diff
@@ -0,0 +1,48 @@
+---
+title: APCU Cache Basics
+description: Understand APC User Cache and its uses within the Pantheon WebOps workflow.
+categories: [platform]
+tags: [cache, workflow]
+reviewed: "2020-10-08"
+---
+
+The **APC User Cache** is a user-accessible PHP cache. PHP is a dynamic language that is compiled on-demand into bytecode at execution time. To improve performance, APCU stores this bytecode so that it can be reused instead of having to be recompiled each time.
+
+Pantheon provides APCU by default across all plans, but the size of the APCU memory cache ([apc.shm_size](https://www.php.net/manual/en/apcu.configuration.php#ini.apcu.shm-size")) varies depending on the service level. See the [Application Containers overview](/application-containers) to learn more about APCU on Pantheon's container architecture.
```

Comment:
All references should be in the form "APCu".

Reply:
All references to "APCU" or "APC User Cache" have been changed to "APCu" in https://github.com/pantheon-systems/documentation/pull/6044/commits/1d273f40730780fc32a1184b8e2705e3482fe298